app/src/main/java/agency/five/codebase/android/movieapp/ui/component/ActorCard.kt

```diff
@@ -0,0 +1,89 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+
+
+data class ActorCardViewState(
+    val imageUrl: String?,
+    val name: String,
+    val character: String,
+)
+
+@Composable
+
+fun ActorCard(
+    actorCardViewState: ActorCardViewState,
+    modifier: Modifier = Modifier,
+){
+    Card(
+        modifier = modifier
+            .height(220.dp)
+            .width(145.dp)
```

Comment:
height and width should be provided through `modifier` from the call site. That way, we can change size of the component from the outside instead of having fixed size for the component.